internal/uvm/network.go

```diff
@@ -90,12 +89,27 @@ func (uvm *UtilityVM) AddNetNS(ctx context.Context, id string) error {
 	if uvm.namespaces == nil {
 		uvm.namespaces = make(map[string]*namespaceInfo)
 	}
-	uvm.namespaces[id] = &namespaceInfo{
+	uvm.namespaces[hcnNamespace.Id] = &namespaceInfo{
 		nics: make(map[string]*nicInfo),
 	}
 	return nil
 }
 
+// AddNetNS adds network namespace inside the guest.
+//
+// If a namespace with `id` already exists returns `ErrNetNSAlreadyAttached`.
+func (uvm *UtilityVM) AddNetNS(ctx context.Context, id string) error {
```

Comment:
I think `AddNetNS` and `AddNetNSByID` would be clearer names, rather than `AddNetNSRaw` and `AddNetNS`.